notebooks/data_processing.ipynb

```diff
@@ -0,0 +1,795 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    api_key = # Enter the API key here.
</pre><p>You can use <a href="https://github.com/theskumar/python-dotenv" target="_blank">https://github.com/theskumar/python-dotenv</a> for any such usages. I'd also suggest to save the elevation at <code>/opt/kdd24</code> once you get it. You can use <code>joblib</code> to make multiple requests at once and make the retrival faster. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/kdd24/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>